.ado/pipelines/templates/jobs-deploy-configuration.yaml

```diff
@@ -44,12 +44,16 @@ jobs:
 
           # Deploy helm chart for ingress-nginx using a custom load balancer ip and resource group (RG in which the Public IP sits in)
           helm upgrade --install ingress-nginx ingress-nginx/ingress-nginx `
-                       --namespace $(ingressNamespace) --create-namespace `
-                       --values src/config/ingress-nginx/values.helm.yaml `
-                       --set controller.service.loadBalancerIP="$aksIngressIp" `
-                       --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-resource-group"="$aksClusterResourceGroup" `
-                       --version "$(ingressNginxVersion)" `
-                       --wait
+                --namespace $(ingressNamespace) --create-namespace `
+                --values src/config/ingress-nginx/values.helm.yaml `
+                --set controller.service.loadBalancerIP="$aksIngressIp" `
+                --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-internal"="true" `
+                --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-internal-subnet"="aks-lb-snet" `
+                --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-pls-create"="true" `
+                --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-pls-name"="aks-ingress-pls" `
+                --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-pls-ip-configuration-subnet"="aks-pl-snet" `
+                --version "$(ingressNginxVersion)" `
+                --wait
```

Comment:
Would it make sense to set the static parameters in `src/config/ingress-nginx` instead? This is really hard to read.

Reply:
hmm we could. But do you think that will make it better? then the "infrastructure" bleeds even more into the config and might be hard for people to find

Reply:
Not sure about that. But still hard to read.. ;)

Reply:
ideally(...) all the subnet names should also not be hardcoded here but sourced from TF output. But I didnt bother :/